composer.json

```
@@ -16,14 +16,14 @@
     ],
     "require": {
         "php": "^8.1",
-        "illuminate/contracts": "^10.0|^11.0",
-        "illuminate/support": "^10.0|^11.0",
-        "illuminate/validation": "^10.0|^11.0",
+        "illuminate/contracts": "^10.0|^11.0|^12.0",
+        "illuminate/support": "^10.0|^11.0|^12.0",
+        "illuminate/validation": "^10.0|^11.0|^12.0",
         "giggsey/libphonenumber-for-php-lite": "^8.13.35"
     },
     "require-dev": {
         "orchestra/testbench": "*",
-        "phpunit/phpunit": "^10.5",
+        "phpunit/phpunit": "^10.5|^11.5.3",
```

```suggestion
        "phpunit/phpunit": "^10.5|^11.5.3|^12.0.1",
```

I would consider updating this to match what's in Laravel 12's [composer.json](https://github.com/laravel/framework/blob/12.x/composer.json#L118) to support PHPUnit `^12`

---

I've merged that in @sweptsquash .. but bear in mind while laravel/framework has phpunit/phpunit ^12.0.1, laravel/laravel skeleton for 12.x still has phpunit/phpunit ^11.5.3 and not 12.